js/libs/keycloak-js/src/keycloak.js

```diff
@@ -450,7 +450,7 @@ function Keycloak (config) {
             url += '&prompt=' + encodeURIComponent(options.prompt);
         }
 
-        if (options && options.maxAge) {
+        if (options && typeof options.maxAge === 'number' && !isNaN(options.maxAge)) {
```

Comment:
Let's not guard against `NaN`, it's up to the user to ensure a correct value is being passed. Also if we do this here we should do it everywhere a number is passed, and that would be a larger refactor I'd rather not get into right now.

Reply:
agree, I have removed the !NaN check and pushed again.